docs/design/garbage-collection.md

```diff
@@ -145,3 +146,74 @@ see: [Bloom filter math](http://pages.cs.wisc.edu/~cao/papers/summary-cache/node
     - Storage node should be able to receive a Delete request from a Satellite
 - Storage node should use the filter from the Delete request to decide which pieces to delete, then delete them
 - Eventually, this service should iterate over a db snapshot instead of being integrated with the data repair checker
+
+## Bloom filters benchmark
+
+Three Bloom filter implementations are considered:
+- **BF1**: Zeebo's bloom filters (github.com/zeebo/sbloom)
+- **BF2**: Willf's Bloom filters (github.com/zeebo/sbloom)
+- **BF3**: Steakknife's Bloom filters (github.com/golang/leveldb/bloom)
+- **BF4**: Custom bloom filter
+
+### Zeebo's bloom filters
```

Comment:
I forgot to comment in the previous review, we could link this titles to the repos for being more exact what ones are referring for.